packages/notifi-react-card/lib/hooks/useNotificationHistory.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  AccountBalanceChangedEventDetails,
+  BroadcastMessageEventDetails,
+  ChatMessageReceivedEventDetails,
+  GenericEventDetails,
+  HealthValueOverThresholdEventDetails,
+  NotificationHistoryEntry,
+} from '@notifi-network/notifi-core';
+import React from 'react';
+
+import { AnnouncementIcon } from '../assets/AnnouncementIcon';
+import { ChatAlertIcon } from '../assets/ChatAlertIcon';
+import { RatioCheckIcon } from '../assets/RatioCheckIcon';
+import { SwapIcon } from '../assets/SwapIcon';
+import { AlertIcon } from '../components/AlertHistory/AlertIcon';
+import { AlertNotificationViewProps } from '../components/AlertHistory/AlertNotificationRow';
+import { formatAmount } from '../utils/AlertHistoryUtils';
+
+export const useNotificationHistory = () => {
```

Comment:
since the hook doesn't actually consume any other hook dependencies, I don't think this should be a hook.

declaring another `.ts` module inside utils is probably cleaner